Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM jenkins/inbound-agent:4.9-1
+
+USER root
+
+RUN apt-get update && apt-get install -y curl rsync wget
+
+WORKDIR /opt/java
+
+RUN wget https://github.com/AdoptOpenJDK/openjdk8-upstream-binaries/releases/download/jdk8u275-b01/OpenJDK8U-jdk_x64_linux_8u275b01.tar.gz
```

Comment:
The Fargate agent uses the exact version of Java we are using now on CentosElasticSlave.
CentosElasticSlave:
[2022-08-24T15:47:09.071Z] + /usr/lib/jvm/jdkredhat-openjdk-1.8.0.275/bin/java -version
[2022-08-24T15:47:09.071Z] openjdk version "1.8.0_275"
[2022-08-24T15:47:09.071Z] OpenJDK Runtime Environment (build 1.8.0_275-b01)
[2022-08-24T15:47:09.071Z] OpenJDK 64-Bit Server VM (build 25.275-b01, mixed mode)

Fargate Agent:
[2022-08-24T15:43:46.143Z] + /opt/java/openjdk-8u275-b01/bin/java -version
[2022-08-24T15:43:46.143Z] openjdk version "1.8.0_275"
[2022-08-24T15:43:46.143Z] OpenJDK Runtime Environment (build 1.8.0_275-b01)
[2022-08-24T15:43:46.143Z] OpenJDK 64-Bit Server VM (build 25.275-b01, mixed mode)